distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -4,10 +4,9 @@ import project.Data.Filter_Condition.Filter_Condition
 import project.Data.List.List
 import project.Data.Map.Map
 import project.Data.Numbers.Integer
-import project.Data.Ordering.Sort_Direction.Sort_Direction
```

Comment:
Lets move `Sort_Direction` from `Ordering` or stop `import all` from Ordering.

Reply:
We have to `import all` from Ordering, because there is a default conversion method for `Comparable.from`. importing all is the only way how to bring that conversion method into the scope. Solved this my moving `Sort_Direction` from `Ordering` module to `Data` module in 78375fa2cacf99e76cd574ce3943911b5c2a0ad8